components/FilterButton.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <v-menu :offset-y="true" :close-on-content-click="false" max-width="250px">
+    <template v-slot:activator="{ on }">
+      <v-btn class="ma-2" icon v-on="on" @click="getFilters">
+        <v-icon>mdi-filter-variant</v-icon>
+      </v-btn>
+    </template>
+    <v-expansion-panels accordion>
+      <v-expansion-panel v-for="item in filters" :key="item.header">
+        <v-expansion-panel-header>{{ item.header }}</v-expansion-panel-header>
+        <v-expansion-panel-content>
+          <v-form v-for="(filter, index) in item.items" :key="index">
+            <v-checkbox
+              v-model="filter.selected"
+              class="my-0"
+              :label="filter.label"
+              :value="filter.value"
+              :true-value="true"
+              :false-value="false"
+              @change="$emit('input', filters)"
```

Comment:
```suggestion
              @change="$emit('change', filters)"
```
A change event makes more sense, since you're changing the filters, I think.